src/application/main.cpp

```diff
@@ -64,13 +65,13 @@ static void parseargs(int argc, char** argv, popsift::Config& config, string& in
         parameters.add_options()
             ("octaves", value<int>(&config.octaves), "Number of octaves")
             ("levels", value<int>(&config.levels), "Number of levels per octave")
-            ("sigma", value<float>()->notifier([&](float f) { config.setSigma(f); }), "Initial sigma value")
+            ("sigma", value<float>()->notifier([&](float f) { config.setSigma(f); }), "Initial sigma value (default 1.6)")
```

Comment:
```suggestion
            ("sigma", value<float>()->notifier([&](float f) { config.setSigma(f); })->default_value(config.getSigma()), "Initial sigma value")
```
using `->default_value()` will automatically print the value when calling `--help`. This is preferable than the raw value as then it becomes difficult to maintain if it ever changes etc.
The same for the others.